src/types/ElectricType.java

```diff
@@ -0,0 +1,44 @@
+package types;
+import pokemon.Pokemon;
+
+
+public class ElectricType extends Pokemon {
```

Comment:
de naam "ElectricType" is misschien niet de beste keus. Het is een Pokemon, want het extend van Pokemon, dus "ElectricPokeon" is een logischere naam.